utils/helpers.js

```diff
@@ -1,7 +1,24 @@
+import { AsyncStorage } from 'react-native'
+import { STORAGE_KEY } from '../constants'
+
 export const generateId = () => Math.random().toString(36).substr(2, 9)
 
 export const formatDate = (isoDate) => {
   const date = new Date(isoDate)
 
   return date.toDateString().substring(4)
 }
+
+export const getHistory = async () => {
+  const data = await AsyncStorage.getItem(STORAGE_KEY)
+  if (data) return JSON.parse(data)
```

Comment:
can you `return data ? JSON.parse(data) : []`

Reply:
Done